pkgs/native_assets_cli/test/api/asset_test.dart

```diff
@@ -14,45 +14,34 @@ void main() {
         name: 'foo',
         file: Uri.file('path/to/libfoo.so'),
         linkMode: DynamicLoadingBundled(),
-        os: OS.android,
-        architecture: Architecture.x64,
       ),
       CodeAsset(
         package: 'my_package',
         name: 'foo3',
         linkMode: DynamicLoadingSystem(Uri(path: 'libfoo3.so')),
-        os: OS.android,
-        architecture: Architecture.x64,
       ),
       CodeAsset(
         package: 'my_package',
         name: 'foo4',
         linkMode: LookupInExecutable(),
-        os: OS.android,
-        architecture: Architecture.x64,
       ),
       CodeAsset(
         package: 'my_package',
         name: 'foo5',
         linkMode: LookupInProcess(),
-        os: OS.android,
-        architecture: Architecture.x64,
       ),
       CodeAsset(
         package: 'my_package',
         name: 'bar',
         file: Uri(path: 'path/to/libbar.a'),
-        os: OS.linux,
-        architecture: Architecture.arm64,
+
```

Comment:
Nit: This blank line here seems out of place?

Reply:
Hm, this is what the formatter does. cc @munificent